source/changelogs/core-2025.7.markdown

```
@@ -1116,16 +1398,22 @@ For a summary in a more readable format:
+[@andersfugmann]: https://github.com/andersfugmann
 [@arturpragacz]: https://github.com/arturpragacz
 [@astrandb]: https://github.com/astrandb
+[@avee87]: https://github.com/avee87
 [@aviadlevy]: https://github.com/aviadlevy
+[@bajansen]: https://github.com/bajansen
 [@balloob]: https://github.com/balloob
 [@bdraco]: https://github.com/bdraco
 [@bieniu]: https://github.com/bieniu
+[@bob-laz]: https://github.com/bob-laz
 [@bouwew]: https://github.com/bouwew
 [@bramkragten]: https://github.com/bramkragten
```

markdownlint-cli2 (0.17.2)

Link and image reference definitions should be needed
Duplicate link or image reference definition: "@bramkragten"

(MD053, link-image-reference-definitions)